tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -41,7 +41,25 @@
 <template is="dom-if" if="[[_dataNotFound]]">
   <div style="max-width: 540px; margin: 80px auto 0 auto;">
     <h3>No profile data was found.</h3>
-    <p>The dashboard is internal only; coming soon!</p>
+    <p>To collect a profile, you need to run your model on Google Cloud TPUs and
+    capture the trace information while your model is running. You may want to
+    check out the
+    <a href="https://github.com/tensorflow/tensorboard/blob/1.6/tensorboard/plugins/profile/README.md">README</a>
+    and perhaps the
+    <a
+     href="https://cloud.google.com/tpu/docs/cloud-tpu-tools"
+    >tutorial</a> on how to use the
+    <a href="https://pypi.python.org/pypi/cloud-tpu-profiler">cloud-tpu-profiler</a>.
+    </p>
+    <p>
+    If you’re new to TPUs, and want to find out how
```

Comment:
Optional nit: I've been trying to train myself personally to avoid using "you" in documentation. I also try to go so far as to figure out prose that avoids saying "one" in its place. Not worth worrying about now. Something to keep in mind.